docs/sections/contributor_guide/documentation.rst

```diff
@@ -1,4 +1,8 @@
 Documentation
 =============
 
-TBD
+WIP
+
+* Ensure that the ``make html`` command completes with no errors or warnings.
+* Do not manually wrap lines. Insert newlines only as needed to achieve correctly formatted HTML, and let HTML handle wrapping long lines.
+* Indent ``.. code::`` directives 2 spaces per nesting level to achieve the required alignment. For example, 0 spaces for code blocks that should be aligned with text on the left margin, 2 spaces for code blocks that should align under bulleted or numbered list text. Indent actual code 2 **extra** spaces under the ``.. code::`` directive.
```

Comment:
This will get fleshed out later with information on adding Sphinx packages to the development shell, running `make html` in the `docs/` directory, previewing docs on your local system, etc, but I wanted to start capturing a few things so that we can be consistent in our formatting.